assets/src/scss/layout/navbar/_burger-menu.scss

```diff
@@ -12,14 +12,14 @@
   top: 0;
   left: 0;
   inset-inline-start: -100%;
+  transition: inset-inline-start 0.2s ease-out;
```

Comment:
If you add this attribute, you'll see a weird burger menu behavior whenever you resize the browser. That's why wasn't added by its default state.

Reply:
Uploading Screen Recording 2022-04-13 at 12.49.49.mov…

Reply:
https://user-images.githubusercontent.com/77975803/163220335-976cedbf-432f-4f39-90a3-6d226905ad10.mov

Reply:
I agree that doesn't look great, but that's only a tiny fraction of cases, in case someone switches from portrait to landscape. The switching animation itself probably makes this unnoticeable.

As for resizing the window, that's also a very rare event. So I'm not sure if it's worth not having a nice animation in all other cases.

Reply:
But let me check if I can fix it :eyes:

Reply:
But if you see this is not a big problem, then we could add it.

Reply:
This seems to work, didn't find any drawbacks yet :crossed_fingers:  https://github.com/greenpeace/planet4-master-theme/pull/1673/commits/bebd164f118c2d640a2e2e05e9f1b623cff937de

Reply:
I changed this to use `:not(.open)`, it needs less rules. https://github.com/greenpeace/planet4-master-theme/blob/377053af1ba308a97dd4bcb009e4be7a86ccb8c6/assets/src/scss/layout/navbar/_burger-menu.scss#L20-L22